contracts/src/CrossChainApplications/NativeTokenBridge/ERC20TokenSource.sol

```diff
@@ -75,48 +66,6 @@ contract ERC20TokenSource is
         erc20ContractAddress = erc20ContractAddress_;
     }
 
-    /**
-     * @dev See {ITeleporterReceiver-receiveTeleporterMessage}.
-     *
-     * Receives a Teleporter message and routes to the appropriate internal function call.
-     */
-    function receiveTeleporterMessage(
-        bytes32 senderBlockchainID,
-        address senderAddress,
-        bytes calldata message
-    ) external nonReentrant {
```

Comment:
we mgiht want to add reentrancy guard to `TeleporterUpgradeable.receiveTeleporterMessage`

Reply:
Yes, I think we should

Reply:
created issue https://github.com/ava-labs/teleporter/issues/213